src/components/person_card/styles.css

```
@@ -0,0 +1,30 @@
+.container {
+    display: grid;
+    grid-template-columns: 1.5fr 3.5fr;
+    border: 2px solid #3f51b58e;
```

Change border color: simple and sobre.
```suggestion
    border: 2px solid #ececed;
```